src/main/java/com/github/dcysteine/neicustomdiagram/generators/gregtech5/oreprocessing/GregTechOreProcessing.java

```diff
@@ -46,7 +47,8 @@ public final class GregTechOreProcessing implements DiagramGenerator {
             OrePrefixes.oreSmall,
             OrePrefixes.orePoor,
             OrePrefixes.oreEndstone,
-            OrePrefixes.oreEnd);
+            OrePrefixes.oreEnd,
+            OrePrefixes.rawOre);
```

Comment:
We're now passing in the raw ore separately, so I think we should remove it from here, to avoid redundancy.